app/src/androidTest/java/org/mozilla/focus/integration/IWebViewExecuteJavascriptTest.kt

```diff
@@ -27,6 +27,7 @@ import org.junit.runner.RunWith
 import org.mozilla.focus.R
 import org.mozilla.focus.MainActivity
 import org.mozilla.focus.browser.BrowserFragment
+import org.mozilla.focus.ext.evalJS
```

Comment:
is this import being used? the body isn't changed...

Reply:
Heh. Yes. The diff is confusing!

It is used here:
https://github.com/mozilla-mobile/firefox-tv/blob/f84f8f6ff98dd7b9a2b8b5eb97b07bede9ce8224/app/src/androidTest/java/org/mozilla/focus/integration/IWebViewExecuteJavascriptTest.kt#L79-L80

Previously this was a method on IWebView. Now it's an extension function on EngineView. Extension functions need to be imported if they are in a different package.

https://kotlinlang.org/docs/reference/extensions.html